Business/Services/CoursesService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Business.DTOs;
+using Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Business.Services;
+public class CoursesService {
+    private UMSDatabaseContext _context;
+
+    public CoursesService(UMSDatabaseContext context) {
+        _context = context;
+    }
+
+    public SemestersAndSpecializationsResponse GetSemestersAndSpecializations() {
+        SemestersAndSpecializationsResponse SemestersAndSpecializations = new SemestersAndSpecializationsResponse();
```

Comment:
local variables are named using the camelCase naming convention, check in other places to make sure you respect the convetion :)